### ISSUE TITLE
Learn Accessibility - Animation and motion content: "used" should be "use" in correct answer response

### ISSUE THREAD

**Issue by MarcSolo** [GoogleChrome/web.dev#8788]
### What page(s) need to be updated? 
Animation and motion

Provide a list of URLs: https://web.dev/learn/accessibility/motion/

### Why is this update needed? 
To improve accuracy and readability of content

Current: "Yes! This media query allows you to decide how much motion to **used** based on a user's Display settings. These settings differ across operating systems, so consider implementing choice for motion in addition to this media query."

Expected: "Yes! This media query allows you to decide how much motion to **use** based on a user's Display settings. These settings differ across operating systems, so consider implementing choice for motion in addition to this media query."
